.gitignore

```diff
@@ -11,3 +11,4 @@ venv
 tests/integration/prometheus-tester/lib/charms/prometheus_k8s/v0/prometheus_scrape.py
 promql-transform
 promql-transform*
+cos-tool*
```

Comment:
We had this ignore ever since `promql-transform` was introduced. 

Moving the download would require a bunch of hacking around `class CosTool` and the path lookup, which is a bad tradeoff.